test/integration/running/test_clean.py

```diff
@@ -1,36 +1,47 @@
 import os
 
-from utils import get_instance_id, get_stateboard_name
-from utils import DEFAULT_CFG, DEFAULT_DATA_DIR, DEFAULT_LOG_DIR, DEFAULT_RUN_DIR
 from utils import write_conf
 from utils import check_instances_running
 from utils import check_instances_stopped
 
 
-CARTRIDGE_CONF = '.cartridge.yml'
-
 FILES_TO_BE_DELETED = ['log', 'workdir', 'console-sock', 'notify-sock', 'pid']
```

Comment:
For constants it's better to use tuples: ``('log', 'workdir', 'console-sock', 'notify-sock', 'pid')``